components/link/index.css

```diff
@@ -10,55 +10,95 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-@import "../vars/css/components/spectrum-link.css";
+.spectrum-Link {
+  /* Colors */
+  /* Primary */
+  --spectrum-link-text-color-primary-default: var(--spectrum-accent-content-color);
+  --spectrum-link-text-color-primary-hover: var(--spectrum-accent-content-color-hover);
+  --spectrum-link-text-color-primary-active: var(--spectrum-accent-content-color-down);
+  --spectrum-link-text-color-primary-focus: var(--spectrum-accent-content-color-key-focus);
 
-.spectrum-Link--sizeS {
-  @remapvars {
-    find: /--spectrum-link(.*)-s-/;
-    filter: color;
-    replace: --spectrum-link$1-;
-  }
+  /* Secondary */
+  --spectrum-link-text-color-secondary-default: var(--spectrum-neutral-content-color-default);
+  --spectrum-link-text-color-secondary-hover: var(--spectrum-neutral-content-color-hover);
+  --spectrum-link-text-color-secondary-active: var(--spectrum-neutral-content-color-down);
+  --spectrum-link-text-color-secondary-focus: var(--spectrum-neutral-content-color-key-focus);
+
+  --spectrum-link-text-color-white: var(--spectrum-white);
+  --spectrum-link-text-color-black: var(--spectrum-black);
 }
 
-.spectrum-Link--sizeM {
-  @remapvars {
-    find: /--spectrum-link(.*)-m-/;
-    filter: color;
-    replace: --spectrum-link$1-;
+/* Windows high contrast mode */
+@media (forced-colors: active) {
+  &--staticBlack {
+    --highcontrast-link-text-color-black: ButtonFace;
```

Comment:
I think this should be ButtonText instead of ButtonFace. Buttonface does not seem to be a text color. Here is our demo that was tested on a PC in dark mode:

![WHCM-dark-pc](https://user-images.githubusercontent.com/25614178/185699391-e09e894e-a195-416c-b5ba-1b550a925a04.png)

Reply:
Yeah, these were the values already on the branch. I forgot to reference that demo.